opt/build/onnxruntime/dockerfile.tmpl

```diff
@@ -0,0 +1,36 @@
+#----------------------------------------------------------------------------------------------
+{% if REDIS_ARCH == 'x64' %}
+{% if REDIS_GPU is defined %}
+FROM nvidia/cuda:11.0-cudnn8-devel-ubuntu18.04
```

Comment:
maybe allow user to define their cuda version?
we use 11.0
might want to compile it on newer versions